### PR TITLE
sql: update statistics

### PR DESCRIPTION
**Pull request by lilin90**
@fipped @CaitinChen PTAL
Via: https://github.com/pingcap/docs-cn/pull/657

### CODE REVIEW

File: sql/statistics.md [hunk 1]
```diff
@@ -125,4 +127,26 @@ Syntax:
 ```sql
 DROP STATS TableName
 > The statement deletes statistics of all the tables in `TableName`。
```
Comment (fipped) [inline sql/statistics.md]: `。` should be `.`.

File: sql/statistics.md [hunk 2]
```diff
@@ -125,4 +127,26 @@ Syntax:
...
+
+### Import statistics
+
+Generally, the imported statistics refer to the JSON file obtained using the export interface.
+
+Syntax:
+
+```
+LOAD STATS 'file_name'
+> `file_name` is the file name of the statistics to be imported
```
Comment (fipped) [inline sql/statistics.md]: Miss `.` in the end.

File: sql/statistics.md [hunk 3]
```diff
@@ -125,4 +127,26 @@ Syntax:
...
+```
+
+### Import statistics
+
+Generally, the imported statistics refer to the JSON file obtained using the export interface.
+
+Syntax:
+
+```
+LOAD STATS 'file_name'
```
Comment (CaitinChen) [inline sql/statistics.md]: 'file_name' -> `file_name`

**Comment by lilin90** [comment]
@fipped Thanks! PTAL again

**Comment by fipped** [comment]
LGTM

**Pull request merged**